public/splash.html

```diff
@@ -4,51 +4,80 @@
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
 
+    <script>
+      if (typeof module === 'object') {
+        // unset module
+        window.module = module;
+        module = undefined;
+      }
+    </script>
+
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/bodymovin/4.13.0/bodymovin_light.min.js"></script>
```

Comment:
I'm not familiar with this package and will take a look at it, but if we want to use a third party library here, I suggest we:

1. Include the [npm package](https://www.npmjs.com/package/bodymovin) in our package.json instead.
2. Change this script tag to use the npm package.

Reply:
Bodymovin is pretty great, but I'm not sure it's necessary.

Reply:
@jeroenptrs My assumption (without having tried yet) is that bodymovin will let us have an animation that's more than just something spinning in a circle.  So it might depend on what the actual animation we're going to use is.  (Unless you have a specific alternative to bodymovin that makes sense, I'm definitely interested in other feedback here.)

Reply:
Yes, bodymovin is actually a powerful tool. You might have heard of lottie for mobile - well this is basically it for web. Even the npm package is called lottie-web. 
In the future, this will allow very, very complex animations to run in a very lightweight and resolution-independant way. Also, the design teams should love this addition as it is “the thing” right now.

I agree with the idea of using npm package, however if we were to do it that way, we’d have to change rendering to go through some templating engine such as ejs or handlebars - simply to inject something from node_modules.

Maybe you know of an easier way?

Btw lottie-web should eventually definitely go into package.json as I see ot commonly used

Reply:
I'm working on another branch that should allow us to use npm packages from these HTML files by compiling through webpack.  With that said, it's not ready yet, and so I'm not worried about making it work quite yet.

Reply:
I just tried this branch locally, and it's a nice improvement.  I don't think we should merge it with a dependency on a remote file though.  Two ideas are:

1. install the npm package, then copy the JS from `node_modules` into `build` as part of the webpack compilation,
2. download the JS file and directly save it to the `public` folder.

For both of these suggestions, we would have to update the `script` tag to reference the local file instead.

@jeroenptrs What do you think about this?  I know you had a separate opinion about not using bodymovin, which may be more valid.

Reply:
@mhuggins Not really, my concern was about adding another dependency that will only be used for one animation. Really like your idea!

Reply:
Sounds good! I'll update the script file to reference it in the `public` folder and, just to make things a bit simpler, add `public/animations` as eslint exceptions (2/3 files are exceptions, so don't see why not)